### PR TITLE
fix excerpt in blog

### PR DESCRIPTION
**Pull request by Steph4104**
get value of french version and use better var in the form

fix issue #1825 
fix issue Phanoix/gcconnex-help#63

### CODE REVIEW

File: mod/wet4/views/default/forms/blog/save.php [hunk 1]
```diff
@@ -29,6 +29,8 @@
 $json_desc = json_decode($vars['description']);
 $json_exc = json_decode($vars['excerpt']);
 
+error_log($vars['excerpt']);
```
Comment (pandurx) [inline mod/wet4/views/default/forms/blog/save.php]: This should probably be removed

File: mod/wet4/views/default/forms/blog/save.php [hunk 2]
```diff
@@ -40,10 +42,12 @@
 }
 
 if ( $json_exc ){
-  $excerpt2 = $json_desc->fr;
-  $excerpt = $json_desc->en;
+  $vars['excerpt2'] = $json_exc->fr;
+  $vars['excerpt'] = $json_exc->en;
 }
 
+error_log('fr '.$excerpt2." en ".$excerpt);
```
Comment (pandurx) [inline mod/wet4/views/default/forms/blog/save.php]: This as well

File: mod/wet4/views/default/forms/blog/save.php (hunk 1 above)
Comment (Steph4104) [reply mod/wet4/views/default/forms/blog/save.php]: fix

File: mod/wet4/views/default/forms/blog/save.php (hunk 2 above)
Comment (Steph4104) [reply mod/wet4/views/default/forms/blog/save.php]: fix

**Pull request merged**